app/Services/WatchListService.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Services;
+
+use App\Comments;
+use App\User;
+use App\WatchList;
+
+class WatchListService
+{
+    /**
+     * Find all movies from the DatabaseTable.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function findAll($user)
+    {
+        return WatchList::with('movie')->where('user_id', $user->id)->get();
+    }
+
+    /**
+     * adds new movie in watchlist.
+     *
+     * @param int $movie_id
+     * @param User $user
+     * @return Comments
+     */
+    public function create(int $movie_id, User $user): WatchList
+    {
+        $watchList['user_id'] = $user->id;
+        $watchList['movie_id'] = $movie_id;
+        return WatchList::create($watchList);
+    }
+
+    /**
+     * Markes movie as watched
+     *
+     * @param int $id
+     * @return Comments
+     */
+    public function markAsWatched(int $id)
+    {
+        return WatchList::where('id', $id)->update(array('watched' => 1));
```

Comment:
array mozes da napises i samo sa []
```suggestion
        return WatchList::where('id', $id)->update(['watched' => 1]);
```